packages/core/package.json

```diff
@@ -4101,8 +4101,8 @@
     },
     "dependencies": {
         "@amzn/codewhisperer-streaming": "file:../../src.gen/@amzn/codewhisperer-streaming",
-        "@aws-sdk/client-cognito-identity": "3.490.0",
-        "@aws-sdk/client-lambda": "3.385.0",
+        "@aws-sdk/client-cognito-identity": "^3.637.0",
+        "@aws-sdk/client-lambda": "^3.637.0",
         "@aws-sdk/client-sso": "^3.342.0",
         "@aws-sdk/client-sso-oidc": "^3.574.0",
         "@aws-sdk/credential-provider-ini": "3.46.0",
```

Comment:
These sdk v3 packages are drifting very far from each other. That's probably why package-lock.json now has 6k lines of workarounds